### ISSUE TITLE
how to prevent eslint linting .eslintrc.js(yep, with extension '.js')

### ISSUE THREAD

**Issue by StinsonZhao** [microsoft/vscode-eslint#831]
I added `.eslintrc.js` as my eslint config file, and then added script `"lint" : "eslint src --ext .ts,.tsx,.js,.jsx"` in `package.json`.

```js
// .eslintrc.js
module.exports = { // at the first, a warning: "File ignored by default.  Use ..."
  // config
};
```

If I use `yarn run lint`, everything is ok. But if I open `.eslintrc.js`, a warning will appear, which is "File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern '!<relative/path/to/filename>'") to override." I think it was been emitted by 'vscode-eslint'.

So, how can I hide this warning? And I don't want to use the `quiet` option to hide all warnings

**Comment by dbaeumer** [comment]
All I can do is to add another setting. The problem is that other wanted to see that message since it is usually printed in the terminal as well.

**Comment by StinsonZhao** [comment]
> All I can do is to add another setting. The problem is that other wanted to see that message since it is usually printed in the terminal as well.

I see. But I want the **vcode-eslint**'s behavior is consistent with my **lint** script, maybe according to some certain mechanisms.

**Comment by StinsonZhao** [comment]
What do you think:

If **vscode-eslint** detects a script called `lint` in package.json, it will use the `lint` script to run the lint task. And further to say, this mechanism can be configurable.

**Comment by dxy2233** [comment]
很多脚手架工具配置eslint使用的的是`.js`文件
希望尽快解决！

**Comment by dbaeumer** [comment]
This got addressed. There is now a setting that can be used to enable warnings for ignored files. It is `eslint.onIgnoredFiles`.
